app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="container">
+  <div class="card">
+    <div class="card-header">
+      <h2>Edit Profile<% resource_name.to_s.humanize %></h2>
+    </div>
+    <div class="card-body">
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+        <%= render "devise/shared/error_messages", resource: resource %>
+
+        <div class="field form-group mt-4 mb-4">
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: "form-control" %>
+        </div>
+
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+        <% end %>
+
+        <div class="field form-group mb-2">
+          <%= f.password_field :password, autocomplete: "new-password", placeholder: "New password", class: "form-control" %>
+          <% if @minimum_password_length %>
+            <em><%= @minimum_password_length %> characters minimum</em>
+          <% end %>
+        </div>
+
+        <div class="field form-group mb-4">
+          <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm new password", class: "form-control" %>
+        </div>
+
+        <div class="field form-group mb-2">
+          <%= f.password_field :current_password, autocomplete: "current-password", placeholder: "Current password", class: "form-control" %>
+          <i>(we need your current password to confirm your changes)</i>
+        </div>
+
+        <div class="field form-group mb-4">
+          <%= f.text_field :name, autocomplete: "name", placeholder: "New name?", class: "form-control" %>
+        </div>
+
+        <div class="actions">
+          <%= link_to "Back", :back, class: "btn btn-secondary" %>
```

Comment:
![image](https://user-images.githubusercontent.com/48946963/125810522-be37cf9c-3712-486c-8ac8-3fb136486ffd.png)

Reply:
Hmm I don't get this display problem. However, I've deleted it because of weird behaviour and also its unnecessity.